src/newSeed/stage3.ts

```diff
@@ -75,13 +75,15 @@ export class Stage3 extends BaseStage {
     } else {
       // Check the token distribution
       let totalDistribAmount = BigNumber.from("0");
-      this.seedConfig.tokenDetails.tokenDistrib.forEach((tokenDistrb: { category: string, amount: string, lockup: number }) => {
-        if (!tokenDistrb.category) {
-          message = "Please enter a value for Category";
-        } else if (!tokenDistrb.amount || tokenDistrb.amount === "0") {
-          message = `Please enter a number greater than zero for Category ${tokenDistrb.category} Amount`;
-        } else if (!(tokenDistrb.lockup > 0)) {
-          message = `Please enter a number greater than zero for Category ${tokenDistrb.category} Lock-up`;
+      this.seedConfig.tokenDetails.tokenDistrib.forEach((tokenDistrb: { stakeHolder: string, amount: string, cliff: number, vest: number }) => {
+        if (!tokenDistrb.stakeHolder) {
+          message = "Please enter a value for Stakeholder";
+        } else if (!tokenDistrb.amount) {
+          message = `Please enter a value for Category ${tokenDistrb.stakeHolder} Amount`;
```

Comment:
Instead of "Category ${tokenDistrb.stakeHolder}"  should just be "Stakeholder"

Reply:
Hmm, actually, just make it "${tokenDistrb.stakeHolder}"

Reply:
I just worry that stakeHolder can be 30 chars, which is long for such a message....

Reply:
I agree, stakeholder might be long making the message even longer